### PR TITLE
Feat: cart page added

### PR DESCRIPTION
**Pull request by arjun-computer-geek**
cart page created

### CODE REVIEW

**Review by Megha-Joshi — APPROVED** [review]
Looks good to me. Can be merged!

**Review by Shailseen — APPROVED** [review]
looks good to me. add screenshots as well. you can go for merge.

**Review by Shubhamkr2610 — APPROVED** [review]
Code looks clean .Nice work. Everything looks good to me. You can merge.

**Review by aryanandarnav13 — COMMENTED** [review]
Good to go

File: css/cart.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
+@import url('./style.css');
+
+.cart-page {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 2em 4em;
```
Comment (aayushjain-code) [inline css/cart.css:7]: try to use rem instead of em,

**Pull request merged**